### PR TITLE
Create quaternaire

### PR DESCRIPTION
**Pull request by flosigime**
Hello, 

I propose a new csl style for the french journal of Quaternary studies. 

Regards

### CODE REVIEW

File: quaternaire.csl [hunk 1]
```diff
@@ -0,0 +1,379 @@
...
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" default-locale="fr-FR">
+  <info>
+    <title>Quaternaire</title>
+    <id>http://www.zotero.org/styles/quaternaire</id>
+    <link rel="self" href="http://www.zotero.org/styles/quaternaire"/>
+    <author>
+      <name>Florent Mercey</name>
+      <email>florent.mercey@inrap.fr</email>
+    </author>
+    <Contributor>
```
Comment (adam3smith) [inline quaternaire.csl]: lowercase the C

File: quaternaire.csl [hunk 2]
```diff
@@ -0,0 +1,379 @@
...
+      <name>François Capron</name>
+      <email>f.capron37@gmail.com</email>
+    </contributor>
+    <category citation-format="author-date"/>
+    <category field="Geology"/>
```
Comment (adam3smith) [inline quaternaire.csl]: and lowercase geology

Comment (flosigime) [reply quaternaire.csl]: thanks !

**Comment by flosigime** [comment]
strange behaviour with the "(Dir) -". No comma after that in my zotero locally...

**Comment by flosigime** [comment]
so strange. It doesn't happens on https://editor.citationstyles.org/codeEditor/

**Comment by rmzelle** [comment]
Thanks! Did you base this on an existing CSL style? If so, we'd like to know which one, so we can add a "template" link (see https://github.com/citation-style-language/styles/wiki/Style-Requirements#8---template-link)

> strange behaviour with the "(Dir) -". No comma after that in my zotero locally...

Can you elaborate a bit on the issue you're seeing? Can you e.g. give the full citation, as rendered by Zotero and by https://editor.citationstyles.org/codeEditor/?

**Comment by flosigime** [comment]
> Thanks! Did you base this on an existing CSL style? If so, we'd like to know which one, so we can add a "template" link (see https://github.com/citation-style-language/styles/wiki/Style-Requirements#8---template-link)

Yes. The base is the revue-archeologique-du-centre-de-la-france style

> 
> > strange behaviour with the "(Dir) -". No comma after that in my zotero locally...
> 
> Can you elaborate a bit on the issue you're seeing? Can you e.g. give the full citation, as rendered by Zotero and by https://editor.citationstyles.org/codeEditor/?

looking for it, but things have changed...

File: quaternaire.csl [hunk 3]
```diff
@@ -27,8 +27,9 @@
         <name-part name="family" text-case="uppercase"/>
         <name-part name="given" text-case="uppercase"/>
       </name>
+      <label form="short" prefix=" (" suffix=")"/>
       <substitute>
-        <text macro="editor" font-weight="bold" suffix=" (Dir.) - "/>
+        <namves variable="editor"/>
```
Comment (flosigime) [inline quaternaire.csl]: and i absolutely need the suffix (Dir.) - for the editors. Deleting this, and i have (Ed.) which is impossible...

Comment (flosigime) [reply quaternaire.csl]: error in namves

**Comment by flosigime** [comment]
> > strange behaviour with the "(Dir) -". No comma after that in my zotero locally...
> 
> Can you elaborate a bit on the issue you're seeing? Can you e.g. give the full citation, as rendered by Zotero and by https://editor.citationstyles.org/codeEditor/?

with you code, rmzelle, on code editor : 

HANCKÉ B., RHODES M. & THATCHER M. (Dir.) - 2007 - Beyond Varieties of Capitalism: Conflict, Contradiction, and Complementarities in the European Economy. Oxford University Press, Oxford and New York, NY.

here, on github

HANCKÉ B., RHODES M. & THATCHER M. (Dir.) - , 2007 - Beyond varieties of capitalism: Conflict, contradiction, and complementarities in the European economy. Oxford University Press, Oxford and New York.

but i'm a bit lost between the differents changes, yours rmzelle, and those made by sebastian...

**Comment by adam3smith** [comment]
Have a look at the sample citations now -- I think these are correct and better in line with both the style guide and examples printed in the journal than what you had before. 
The extra comma in the sample references earlier was due to the fact that the software used to create citations here is slightly different from that in the the editor and Zotero, so there are sometimes small differences.

**Comment by flosigime** [comment]
> Have a look at the sample citations now -- I think these are correct and better in line with both the style guide and examples printed in the journal than what you had before.

Yes thanks a lot ! It seems really nice. With an uppercase to dir (Dir.), it will be perfect.  
(Sorry, i didn't find how to create a commit...Github is weird to me....)

> The extra comma in the sample references earlier was due to the fact that the software used to create citations here is slightly different from that in the the editor and Zotero, so there are sometimes small differences.

Ok for the sample. I thought it could be something like this. It worked on zotero and everywhere else. I was not really worried...

**Comment by flosigime** [comment]
yes i Did find how to create a commit :-D

**Comment by adam3smith** [comment]
Where are you getting the capitalized dir. from? I'm seeing
image
in the instructions for authors and consistently lowercase dir. in the journal

**Comment by flosigime** [comment]
I did it. in the last commit : 

> HANCKÉ B., RHODES M. & THATCHER M. (Dir.), 2007 - Beyond varieties of capitalism: Conflict, contradiction, and complementarities in the European economy. Oxford University Press, Oxford and New York.

> MARES I., 2001 - Firms and the welfare state: When, why, and how does social policy matter to employers? In P. A. Hall et D. Soskice (Dir.), Varieties of capitalism. The institutional foundations of comparative advantage. Oxford University Press, New York, 184‑213.

**Comment by adam3smith** [comment]
I understand that you changed it in the style code. I'm asking why you did, because that's not what I'm seeing in the journal and its instructions.

**Comment by flosigime** [comment]
Because if in english the term editor is used, in french, this is the job of the "Directeur de la publication"...
I asked the question to the editor of Quaternaire. They have never though about it, and in fact, they often use "éditeur", a direct translation of editor. But a mistake. 
Anyway, indeed, they will use ed and eds, even if it's an anglicism. 
I make the correction.

**Comment by adam3smith** [comment]
I would actually have been fine with Dir. -- but why capitalized? It's lowercased in the instructions for authors and all articles I liked at.

**Comment by flosigime** [comment]
yes, you're right. No capital...

**Comment by flosigime** [comment]
Thank you !!!

**Pull request merged**